src/StructuredLogger/BinaryLogger/Postprocessing/BinlogRedactor.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Build.Logging;
+using Microsoft.Build.Logging.StructuredLogger;
+using System.IO;
+using System.Threading;
+using Microsoft.Build.SensitiveDataDetector;
+
+namespace StructuredLogger.BinaryLogger.Postprocessing

```

Comment:
I have to confess that the namespace naming is historically kind of a mess in the MSBuildStructuredLog repo. I'm hoping to move towards rationalizing it all a bit.

For this one, how about `Microsoft.Build.Logging.BinaryLogger.Postprocessing`? Or maybe just leave `Microsoft.Build.Logging.BinaryLogger`?

Also, would it be possible to move this type out to StructuredLogViewer.Core perhaps? If we're going to try to avoid having StructuredLogger.dll to have a dependency on the redaction logic.

Reply:
I moved all files in Postprocessing folder to `Microsoft.Build.Logging.StructuredLogger` namespace